tests/functional/client.js

```diff
@@ -60,4 +60,33 @@ describe('Bucket Client tests', function testClient() {
             return done(new Error('Did not fail as expected'));
         });
     });
+
+    it('should get logs from bucketd', done => {
```

Comment:
functional tests are not launched automatically in this repository (see circle.yml)
We should probably open a ticket to do so soon...
We'll need to test this manually with a devmode setup in MetaData before merging